expressions/methods.md

```diff
@@ -196,6 +196,21 @@ class Foo
 
 __Can I call using positional arguments but miss out the first one?__ No. If you use positional arguments they must be the first ones in the call.
 
+## Chaining
+
+Method chaining allows you to chain calls on an object without requiring the method to return its receiver. The syntax to call a method and chain the receiver is `object.>method()`, which is roughly equivalent to `(object.method() ; object)`. Chaining a method discards its original return value.
+
+```pony
+primitive Printer
+  fun print_two_strings(out: StdStream, s1: String, s2: String) =>
+    out.>print(s1).>print(s2)
+    // Equivalent to:
+    out.print(s1)
+    out.print(s2)
+```
+
+Note that the last `.>` in a chain can be a `.` if the return value of the last call matters.
```

Comment:
I think it is important to drive home what this means.

If .>print(s2) then the return value is "out".
If .print(s2) then the return value is None

This becomes particularly important for methods that have a non-None value as the normal return value.

Examples of that case as well would be very good to drive home the point of what is happening.